### PR TITLE
Plastic Surgery Fix's

### PR DESCRIPTION
**Pull request by dragoslava**
Forgot a few things for my plastic surgery PR, including support for shells and robotics repairs

### CODE REVIEW

**Comment by dragoslava** [comment]
!review

**Comment by skull132** [comment]
hnrg why is this an entire file move.

**Comment by dragoslava** [comment]
> hnrg why is this an entire file move.

Because there was no reason to have stuff in two different files for one procedure.

File: code/modules/surgery/facial_surgery.dm [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+		user.visible_message("<span class='warning'>[user]'s hand slips, slicing [target]'s throat wth \the [tool]!</span>" , \
+		"<span class='warning'>Your hand slips, slicing [target]'s throat wth \the [tool]!</span>" )
+		affected.createwound(CUT, 60)
+
+
+/datum/surgery_step/robotics/face/prepare_face
+	allowed_tools = list(
+	/obj/item/weapon/retractor = 100,
+	/obj/item/weapon/material/hatchet/tacknife = 75
```
Comment (MarinaGryphon) [inline code/modules/surgery/facial_surgery.dm]: This does not work, and results in amputation, IIRC.

File: code/modules/surgery/facial_surgery.dm [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+			return 0
+		return target_zone == "mouth"
+
+
+
+
+/datum/surgery_step/generic/prepare_face
+	allowed_tools = list(
+	/obj/item/weapon/retractor = 100,
+	/obj/item/weapon/material/hatchet/tacknife = 75
```
Comment (MarinaGryphon) [inline code/modules/surgery/facial_surgery.dm]: Results in amputation, I believe, due to priority conflicts.

File: code/__defines/mobs.dm [hunk 3]
```diff
@@ -84,7 +84,7 @@
 #define APPEARANCE_EYE_COLOR 0x100
 #define APPEARANCE_ALL_HAIR (APPEARANCE_HAIR|APPEARANCE_HAIR_COLOR|APPEARANCE_FACIAL_HAIR|APPEARANCE_FACIAL_HAIR_COLOR)
 #define APPEARANCE_ALL       0xFFFF
-#define APPEARANCE_PLASTICSURGERY (APPEARANCE_HAIR|APPEARANCE_HAIR_COLOR|APPEARANCE_FACIAL_HAIR|APPEARANCE_FACIAL_HAIR_COLOR|APPEARANCE_GENDER) & ~APPEARANCE_RACE
+#define APPEARANCE_PLASTICSURGERY (APPEARANCE_HAIR|APPEARANCE_HAIR_COLOR|APPEARANCE_FACIAL_HAIR|APPEARANCE_FACIAL_HAIR_COLOR|APPEARANCE_GENDER|APPEARANCE_SKIN|APPEARANCE_EYE_COLOR) & ~APPEARANCE_RACE
```
Comment (MarinaGryphon) [inline code/__defines/mobs.dm]: `#define APPEARANCE_PLASTICSURGERY (APPEARANCE_ALL & ~APPEARANCE_RACE)`? Much shorter, and reinforces the fact that it's everything but species.

File: code/modules/surgery/facial_surgery.dm (hunk 1 above)
Comment (dragoslava) [reply code/modules/surgery/facial_surgery.dm]: https://puu.sh/CkWuI/912b7cfab6.mp4

You might of been targeting the patients head, the plastic surgery method doesnt amputate if you target the mouth

**Pull request merged**